source/docs/software/advanced-controls/geometry/coordinate-systems.rst

```diff
@@ -0,0 +1,20 @@
+Coordinate Systems
+==================
+
+In FRC, there are two main coordinate systems that we use for representing objects' positions.
+
+Field Coordinate System
+-----------------------
+
+The field coordinate system (or global coordinate system) is an absolute coordinate system where a point on the field is designated as the origin. Positive θ is in the counter-clockwise direction, and the x-axis goes through the driver stations.
+
+.. image:: diagrams/coordinate-systems-1.drawio.svg
```

Comment:
In addition to this diagram, it would be useful to overlay the coordinate axes over the 2020 FRC Field (like an image from Pathweaver).